compiler/packages/babel-plugin-react-compiler/src/HIR/HIRBuilder.ts

```diff
@@ -311,6 +311,12 @@ export default class HIRBuilder {
   }
 
   resolveBinding(node: t.Identifier): Identifier {
+    if (node.name === 'fbt') {
```

Comment:
We could also bail out when fbt is not directly imported, but that's not necessary as we only rename locals within functions the compiler processes